README.md

```diff
@@ -45,6 +45,27 @@ It aims to create a customizable, extendable and lightweight environment without
 - [wf-config](https://github.com/WayfireWM/wf-config)
 - [wlroots](https://github.com/swaywm/wlroots)
 
+### Login Provider (optional)
```

Comment:
I also think that it might be worth to say login provider and xwayland dependencies are optional, but highly recommended (after all, except if you don't really know what you're doing, you probably want to have these)

Reply:
> I also think that it might be worth to say login provider and xwayland dependencies are optional, but highly recommended (after all, except if you don't really know what you're doing, you probably want to have these)

Maybe we can use (optional, recommended) 'tag'.

Reply:
Sounds good! I will make the changes here till there's a definite agreement, then I can update the wiki page,